doozer/doozerlib/backend/rebaser.py

```diff
@@ -2164,7 +2167,60 @@ def _map_image_name(name, image_map):
                         sr_file_str = sr_file_str.replace(s, r)
                         if sr_file_str == original_string:
                             self._logger.error(f'Search `{s}` and replace was ineffective for {metadata.distgit_key}')
+                            sr_list.append((s, r))
 
                 # Overwrite file with updated content
                 async with aiofiles.open(f_path, 'w', encoding='utf-8') as sr_file:
                     await sr_file.write(sr_file_str)
+
+            # check if there is already a JIRA issue opened for this component
+            jira_client: JIRA = Runtime.build_jira_client(self._runtime)
+            major, minor = int(self._runtime.group_config.vars.MAJOR), int(self._runtime.group_config.vars.MINOR)
+            jql_query = (
+                f'project = "OCPBUGS" AND '
+                f'summary ~ "Make substitutOOOns work for {major}.{minor} in manifests of {metadata.distgit_key}" AND '
+                f'labels in (art:{major}-{minor}:fix-olm-substitution) AND '
+                f'status not in (Closed, Resolved) '
+            )
+
+            issues = jira_client.search_issues(jql_query)
+            formatted_sr_list = [f"- search: {s_val} \n  replace: {r_val}" for s_val, r_val in sr_list]
+            formatted_output = '\n'.join(formatted_sr_list)
+            is_comp = metadata.distgit_key
+            issue_comp = self.get_issue_component(is_comp)
+            if not issues:
+                self._logger.info(f"a new JIRA issue for {metadata.distgit_key} will be raised")
+                # raise a new JIRA issue
+                fields = {
+                    'project' : {'key': 'OCPBUGS'},
+                    'summary': f'Make substitutions work for {major}.{minor} in manifests of {metadata.distgit_key}',
```

Comment:
This is already defined at line 2181, better define it as a variable to facilitate maintenance